examples/mplot3d/rotate_axes3d_demo.py

```diff
@@ -1,13 +1,21 @@
+'''
+A very simple animation of a rotating 3D plot.
+
+See wire3d_animation_demo for another simple example of animating a 3D plot.
+'''
+
 from mpl_toolkits.mplot3d import axes3d
 import matplotlib.pyplot as plt
-import numpy as np
-
 
 fig = plt.figure()
 ax = fig.add_subplot(111, projection='3d')
+
+# load some test data for demonstration and plot a wireframe
 X, Y, Z = axes3d.get_test_data(0.1)
 ax.plot_wireframe(X, Y, Z, rstride=5, cstride=5)
 
+# rotate the axes and update
 for angle in range(0, 360):
     ax.view_init(30, angle)
     plt.draw()
+    plt.pause(.001)
```

Comment:
@QuLogic You are correct.  `plt.pause()` spins the GUI event loop so the update gets processed.

I don't think we need to worry about the warning in this PR, it is a know issue.